src/common/Input/Input.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Input.css';
+
+const Input = ({
+	value,
+	onChange,
+	placeholder,
+	type = 'text',
+	...otherProps
+}) => {
+	return (
+		<input
```

Comment:
To be semantically correct please wrap this with form and add a label tag for input